### PR TITLE
allow padding mixed 2d and 3d conditionings

### PR DESCRIPTION
**Pull request by jameshurst**
This change allows passing `__call__` a list containing a mix of blended prompts and plain prompts. This lets you use a list of prompts such as `['("spider man", "robot mech").blend(1, 0.8)', 'spider man, robot mech']` so you can easily compare the differences.

The logic in the padding function caused all tensors to end up as 3 dimensional, so on a list of mixed 2/3 dimensional tensors we can simply unsqueeze any 2 dimensional tensors for the same result.

Note: This could probably already be done by invoking `__call__` multiple times and calling `torch.cat` on the results since `__call__` always returns a 3 dimensional tensor due to the unsqueezing already done by the padding function.

### CODE REVIEW

**Review by damian0815 — APPROVED** [review]
very nice, thank you
